tests/PHPStan/Rules/Comparison/data/bug-6213.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Bug6213;
+
+use DOMDocument;
+use DOMText;
+
+class HelloWorld
+{
+	public function sayHello(): void {
+		$document = new DOMDocument('1.0', 'utf-8');
+		$element = $document->createElement('div', 'content');
+
+		// Incorrect! This is DOMNode|null not DOMElement|null
+		// It's also possible to contain a DOMText node, which is not an instance
+		// of DOMElement, but an instance of DOMNode!
+		if ($element->firstChild instanceof DOMText) {
```

Comment:
This test would never fail. The error message is: `Instanceof between DOMElement|null and DOMText will always evaluate to false.`

This test needs to be in ImpossibleInstanceOfRuleTest. Please verify that locally by reverting the stubs update commit. Thanks.

Reply:
I updated the test

It fails without the stub update
<img width="678" alt="image" src="https://user-images.githubusercontent.com/9052536/161225093-3601ae91-233e-4da6-bbaa-cc51f5fe10b5.png">